packages/cloud-cognitive/src/components/APIKeyModal/APIKeyModal.js

```diff
@@ -33,19 +33,26 @@ import uuidv4 from '../../global/js/utils/uuidv4';
 
 const componentName = 'APIKeyModal';
 
+// Default values for props
+const defaults = {
```

Comment:
Are you suggesting we do not use `...defaults` as it is clearer?

Reply:
I couldn't find a neater way :-)